tox.ini

```diff
@@ -14,7 +14,8 @@ commands = flake8 user_util
 [testenv]
 setenv = 
     PYTHONPATH = {toxinidir}
-deps = 
+deps =
+	setuptools 
```

Comment:
What was the reason for adding setuptools here?

Reply:
I saw this error in other PR by bot here https://github.com/openedx/user-util/actions/runs/8019322934/job/21906966665?pr=23 

And this fix was applied in other repos (https://github.com/openedx/edx-django-utils/pull/384/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R73) so I just did the same thing to see if the check will get passed or not. 

The plan was to fix each failing tests one by one.